config/webpack.prod.js

```diff
@@ -11,6 +11,7 @@ module.exports = webpackMerge(commonConfig, {
     devtool: 'source-map',
 
     plugins: [
+        new webpack.optimize.CommonsChunkPlugin({ name: ['main', 'vendor', 'polyfills'], minChunks: Infinity }),
```

Comment:
I was getting a WebpackJsonp not defined error, So I googled and found this https://github.com/webpack/webpack/issues/368. May be the issue is something else, well I tried figuring out but was unsuccessful. Removing that helped to get the Testing framework up.